src/main/features/dashboard/views/claimant.njk

```diff
@@ -15,15 +15,21 @@
     <div class="column-two-thirds">
       <div>
         <h2 class="heading-medium">{{ t('Claim status') }}</h2>
-        <div class="panel panel-border-wide">
-          <p>
-            {{ t('The defendant %s has not responded to your claim. ',
-            { postProcess: 'sprintf', sprintf: [claim.claimData.defendant.name] }) }}
-            {% if claim.eligibleForCCJ %}
-              {{ t('You can request a County Court Judgment (CCJ)') }}
-            {% endif %}
-          </p>
-        </div>
+
+        {% if claim.respondedAt %}
+          <p>{{ t('The defendant %s has responded to your claim. ',
```

Comment:
should there now be a link to download the response for the claimant in the side panel?
(it was missed earlier and currently the support team is forwarding the response to the claimant)

Reply:
I don't know. I just added this basic info as nothing was displayed to claimant when defendant has responded. 

There is quite a lot of different options here ( https://hmcts-cmc-prototype.herokuapp.com/dashboard/v2/claimant?claimant=3 )

I'll merge it as it is (to unblock Pragnesh) and create a separate PR for message when defendant has responded.

Reply:
it is supposed to have the defendant's response on the side panel would be very easy to add it in for the next PR 👍